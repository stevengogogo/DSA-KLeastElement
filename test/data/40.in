10 50000
-2757 -343 -8165 5394 -7848 -2154 -5878 4709 2491 5008
Delete 8
Insert 7 2556
Query 10 10 1
Delete 7
Reverse 1 6
Reverse 7 9
Query 3 3 1
Query 3 4 2
Delete 1
Reverse 5 6
Reverse 2 8
Delete 7
Delete 4
Query 2 4 3
Delete 6
Insert 5 -1481
Delete 2
Reverse 1 3
Reverse 2 4
Delete 1
Reverse 2 2
Delete 3
Insert 1 -3908
Reverse 4 4
Delete 2
Query 1 2 1
Delete 3
Delete 1
Insert 2 9693
Query 2 2 1
Delete 1
Reverse 1 1
Insert 1 -5548
Insert 2 -2878
Insert 1 5442
Query 1 1 1
Reverse 3 4
Delete 3
Delete 1
Insert 1 -7213
Reverse 1 2
Reverse 2 3
Query 3 3 1
Delete 3
Insert 1 -7306
Insert 1 -6907
Query 3 4 2
Delete 4
Reverse 2 3
Delete 3
Reverse 2 2
Delete 2
Delete 1
