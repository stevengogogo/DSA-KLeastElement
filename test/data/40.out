5008
5394
5394
5008
-7848
9693
5442
-7213
-2878
